_includes/author-profile.html

```diff
@@ -1,6 +1,6 @@
 {% include base_path %}
 
-{% if page.author %}
+{% if page.author and site.data.authors[page.author] %}
```

Comment:
What was the thinking behind changing this line? What happens if someone doesn't create an `authors.yml` data file?